odl/tomo/geometry/conebeam.py

```diff
@@ -665,23 +711,28 @@ def __init__(self, apart, dpart, src_radius, det_radius, pitch=0,
 
         # Check and normalize `src_to_det_init`. Detector axes are
         # normalized in the detector class.
-        if np.linalg.norm(src_to_det_init) <= 1e-10:
-            raise ValueError('`src_to_det_init` norm {} too close to 0'
-                             ''.format(np.linalg.norm(src_to_det_init)))
+        if np.linalg.norm(src_to_det_init) == 0:
```

Comment:
I expect issues regarding scales. If somebody works on electron tomography where lengths are in the range of nanometers, and they decide to not (or forget to) normalize the vector, the old check will produce an unreasonable error.

With all these closeness checks on user input my attitude is to let the user figure out what to provide and how to deal with numerical accuracy issues, instead of us imposing some arbitrary bounds.